R/steps-rcmdcheck.R

```diff
@@ -153,11 +153,12 @@ step_rcmdcheck <- function(...,
   #' @param args `[character]`\cr
   #'   Passed to `rcmdcheck::rcmdcheck()`.\cr
   #'   Default for Travis and local runs: `c("--no-manual", "--as-cran")`.\cr
-  #'   Default for Appveyor:
+  #'   Default for Appveyor and GH Actions (Windows):
   #'   `c("--no-manual", "--as-cran", "--no-vignettes",
   #'   "--no-build-vignettes", "--no-multiarch")`.\cr
   if (is.null(args)) {
-    if (isTRUE(ci_on_appveyor())) {
+    if (isTRUE(ci_on_appveyor()) |
```

Comment:
`||` implements a shortcut, the RHS is not evaluated if the LHS is true.

```suggestion
    if (isTRUE(ci_on_appveyor()) ||
```

Reply:
I'm aware of the shortcut - but you are right, `||` is correct here :)